cmd/ipfs/daemon.go

```diff
@@ -208,6 +209,16 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 	// let the user know we're going.
 	fmt.Printf("Initializing daemon...\n")
 
+	bsLogLevel := os.Getenv("BS_CFG_LOG");
+	if  bsLogLevel == "" {
+		panic("BS_CFG_LOG not set, recommended to set to INFO")
+	} else {
```

Comment:
Can you set this to some default that works for you? I'd like to make deploying this on a gateway as simple as possible.

Ideally this includes just asking infra to checkout the branch and build it before running the rest of the normal code (e.g. and not having to run `run-bs-monitor.bash`). I'm sure we can do it, but if we could just build the branch that'd likely be much easier.

Reply:
Updated to 'info' as default. Also removed the 'warn' log level for the rest of the subsystems to be as least disruptive as possible of standing GW configurations.